jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/ServletApiRequest.java

```diff
@@ -73,11 +77,16 @@
 import org.eclipse.jetty.session.ManagedSession;
 import org.eclipse.jetty.util.Fields;
 import org.eclipse.jetty.util.HostPort;
+import org.eclipse.jetty.util.IO;
 import org.eclipse.jetty.util.StringUtil;
 import org.eclipse.jetty.util.URIUtil;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static org.eclipse.jetty.ee10.servlet.ServletContextRequest.INPUT_NONE;
```

Comment:
This constant is imported static, but its sibling `INPUT_STREAM` is not, see code below.
I would either import static them, or (my preferred) remove the import static and use the qualification explicitly.

Reply:
done